__tests__/components/editor/ImportResourceTemplate.test.js

```diff
@@ -80,6 +98,8 @@ describe('<ImportResourceTemplate />', () => {
         createResourceError: ['i am an error for a created object']
       })
     })
+
+    // TODO: test scenario where id token retrieval fails (e.g. expired refresh token)
```

Comment:
captured in #528, though this will almost definitely be broken off into its own ticket

Reply:
I learned from Mike that we can do 

```jest
it.todo('when refresh token is expired')
```